src/data/courseDependenciesData.js

```diff
@@ -929,7 +929,7 @@ const courseDependencyData = (courseSlug) =>
         slug: 'create-an-ecommerce-store-with-next-js-and-stripe-checkout-562c',
         dependencies: {
           react: '>= 17.0.1',
-          next: '>=9.5.5',
+          next: '^10.0.6',
```

Comment:
Colby updated all the lesson to ^10.0.6 - [reference](https://github.com/colbyfayock/space-jelly-store-workshop/commit/40ef796469c741e6ea9744a9dc92f7cfe9e9282f)